pkg/operator/apis/monitoring/doc.go

```diff
@@ -0,0 +1 @@
+package monitoring
```

Comment:
may as well add a basic package level doc comment so this file has a reason to exist, other than the tool requires it.